### PR TITLE
docs: bump docker lnd dev and testing version

### PR DESCRIPTION
**Pull request by RandyMcMillan**

### CODE REVIEW

**Comment by RandyMcMillan** [comment]
Bump version and add links to tags for ease of use.

File: docs/DOCKER.md [hunk 1]
```diff
@@ -110,17 +110,16 @@ to expose RPC ports, use `btcd` or `bitcoind`, or add additional chains.
...
-To test the Docker production image locally, run the following from
-the project root:
+To test the Docker production image locally, run the following from the project root:
 
 ```shell
 ⛰  docker build . -t myrepository/lnd:master
 ```
 
-To choose a specific branch or tag instead, use the "checkout" build-arg.  For example, to build the latest commits in master:
+To choose a specific [branch](https://github.com/lightningnetwork/lnd/tags) or [tag](https://hub.docker.com/r/lightninglabs/lnd/tags?page=1&ordering=last_updated) instead, use the "checkout" build-arg.  For example, to build the latest commits in master:
```
Comment (wpaulino) [inline docs/DOCKER.md:119]: Branches are separate from tags, so that link should point to https://github.com/lightningnetwork/lnd/branches instead?

**Review by carlaKC — APPROVED** [review]
LGTM pending @wpaulino's suggestion.

**Review by kevkevinpal — APPROVED** [review]

File: docs/DOCKER.md [hunk 2]
```diff
@@ -110,17 +110,16 @@ to expose RPC ports, use `btcd` or `bitcoind`, or add additional chains.
...
-To test the Docker production image locally, run the following from
-the project root:
+To test the Docker production image locally, run the following from the project root:
 
 ```shell
 ⛰  docker build . -t myrepository/lnd:master
 ```
 
-To choose a specific branch or tag instead, use the "checkout" build-arg.  For example, to build the latest commits in master:
+To choose a specific [branch](https://github.com/lightningnetwork/lnd/branches) or [tag](https://hub.docker.com/r/lightninglabs/lnd/tags?page=1&ordering=last_updated) instead, use the "checkout" build-arg.  For example, to build the latest commits in master:
```
Comment (wpaulino) [inline docs/DOCKER.md:119]:
```suggestion
To choose a specific [branch](https://github.com/lightningnetwork/lnd/branches) or [tag](https://hub.docker.com/r/lightninglabs/lnd/tags?page=1&ordering=last_updated) instead, use the "checkout" build-arg. For example, to build the latest tagged commit:
```

File: docs/DOCKER.md [hunk 3]
```diff
@@ -110,17 +110,16 @@ to expose RPC ports, use `btcd` or `bitcoind`, or add additional chains.
...
 
 ```shell
-⛰  docker build . --build-arg checkout=v0.8.0-beta -t myrepository/lnd:v0.8.0-beta
+⛰  docker build . --build-arg checkout=v0.13.0-beta.rc5 -t myrepository/lnd:v0.13.0-beta.rc5
```
Comment (wpaulino) [inline docs/DOCKER.md:122]:
```suggestion
⛰  docker build . --build-arg checkout=v0.13.0-beta -t myrepository/lnd:v0.13.0-beta
```

**Comment by RandyMcMillan** [comment]
BEFORE: [2c634bf ](https://github.com/lightningnetwork/lnd/blob/master/docs/DOCKER.md)

AFTER: [6376d78](https://github.com/lightningnetwork/lnd/blob/6376d78b3e4d920e044463c52d4b1e7c25f6295b/docs/DOCKER.md)

**Review by wpaulino — APPROVED** [review]

**Review by Roasbeef — APPROVED** [review]
LGTM 🐠

**Pull request merged**